### PR TITLE
Generate warnings on using Past Tense

### PR DESCRIPTION
**Pull request by avik-pal**
## Preview
Screenshot from 2020-07-14 12-44-11

## Features
1. Checks for the present of Past tense in the commit message
2. Suggests generic Past to Present tense fixes

### CODE REVIEW

File: git_lint_branch/main.py [hunk 1]
```diff
@@ -4,21 +4,23 @@
...
 
+
 @app.command()
 def main(upstream: str):
     """
     Lints the commit history reachable from the current HEAD that is not
     on UPSTREAM (i.e., the current branch).
     """
-    repo = Repository('.git')
+    repo = Repository(".git")
```
Comment (jiviteshjain) [inline git_lint_branch/main.py]: Looks like your editor changes single quotes to double?

Comment (avik-pal) [reply git_lint_branch/main.py]: Yup. It runs `black` on python repos by default. Let me revert this.

Comment (juped) [reply git_lint_branch/main.py]: I happen to prefer single quotes for a specific reason:
```
>>> "this is a string"
'this is a string'
```
they're used in the default `__repr__` for strings. The exception is docstrings where some PEP mandates `"""`, I believe

**Review by juped — CHANGES_REQUESTED** [review]
Has a lot of extraneous formatting changes, should just be code changes. Not big on adding dependencies but I guess spacy is widely useful for this sort of thing, so it's good.

**Comment by avik-pal** [comment]
Sorry about those I have opened a clean PR #15

**Pull request closed without merging**